osidb/filters.py

```diff
@@ -260,6 +260,30 @@ class FlawQLSchema(DjangoQLSchema):
     any field in the model, which is not desirable in this case.
     """
 
+    include = (
+        Flaw,
+        Affect,
+        Tracker,
+        Package,
+        FlawCVSS,
+        FlawAcknowledgment,
+        FlawReference,
+    )
+
+    suggest_options = {
+        Flaw: [
+            "source",
+            "impact",
+            "requires_cve_description",
+            "major_incident_state",
+            "workflow_state",
```

Comment:
The order of the items seems quite random. If there is no rule to sort I would prefer the alphabet then